src/python/pants/backend/python/target_types.py

```diff
@@ -570,8 +570,15 @@ class PexEntryPointsField(StringSequenceField, AsyncFieldMixin):
     alias = "entry_points"
     default = None
     help = (
-        # @TODO: Update
-        "..."
+        "The entry points for each binary, i.e. what gets run when when executing `./my_app.pex.`"
+        "\n\n"
+        "Use a file name, relative to the BUILD file, like `app.py`. You can also set the "
+        "function to run, like `app.py:func`.\n"
```

Comment:
Nit: we don't want to use this `\n` because it breaks soft wrapping. Only use `\n\n` for new paragraphs